license-checker/npm/action.yml

```diff
@@ -0,0 +1,41 @@
+name: Check NPM License Compliance
+description: Check NPM dependencies for license compliance using license-checker
+
+inputs:
+  subdirectory:
+    description: The subdirectory to check dependencies
+    default: .
+    required: false
+  allowed_licenses:
+    description: Allowed license types (semicolon separated)
+    default: MIT;Apache;ISC;BSD;Python;CC;Public Domain;BlueOak;Unlicense
+    required: false
+  clarifications_file:
+    description: File to clarify package license
+    default: ../.github/clarification_file_license.json
```

Comment:
@joshuagibeonm this file later will be supplied on the repository that implement this action?
Can you share one example of this?